### ISSUE TITLE
pricing of gcp t2a-standard

### ISSUE THREAD

**Issue by KivraChristoffer** [infracost/infracost#2420]
```
resource "google_compute_instance" "runners" {
  count        = 3
  name         = "runner-${format("%02d", count.index + 1)}"
  machine_type = "t2a-standard-1"
  zone         = "europe-west4-a"
  project      = google_project.project.project_id
  tags = ["gh-runner", "github-runner"]

  boot_disk {
    initialize_params {
      size  = 100
      image = "ubuntu-os-cloud/ubuntu-minimal-2204-jammy-arm64-v20230420"
      labels = {
        my_label = "value"
      }
    }
  }

  network_interface {
    network = "default"
    access_config {
      // Ephemeral public IP
    }
  }
  metadata = {
    foo = "bar"
  }
  depends_on = [
    google_project.project
  ]
}
```

would give
```
Name                                                       Monthly Qty  Unit   Monthly Cost 
                                                                                             
 google_compute_instance.runners[0]                                                          
 ├─ Instance usage (Linux/UNIX, on-demand, t2a-standard-1)          730  hours         $0.00 
 └─ Standard provisioned storage (pd-standard)                      100  GB            $4.40 
                                                                                             
 google_compute_instance.runners[1]                                                          
 ├─ Instance usage (Linux/UNIX, on-demand, t2a-standard-1)          730  hours         $0.00 
 └─ Standard provisioned storage (pd-standard)                      100  GB            $4.40 
                                                                                             
 google_compute_instance.runners[2]                                                          
 ├─ Instance usage (Linux/UNIX, on-demand, t2a-standard-1)          730  hours         $0.00 
 └─ Standard provisioned storage (pd-standard)                      100  GB            $4.40 
                                                                                             
 OVERALL TOTAL                                                                        $13.20 
──────────────────────────────────
5 cloud resources were detected:
∙ 3 were estimated, all of which include usage-based costs, see https://infracost.io/usage-file
∙ 2 were free:
  ∙ 1 x google_project
  ∙ 1 x google_project_service
```
gcp reports each vm kosts $41.93
![Screenshot from 2023-04-25 09-55-59](https://user-images.githubusercontent.com/75297970/234212029-d0b485b6-2794-4b79-bf68-2975364a4c5d.png)

**Comment by aliscott** [comment]
I'm not seeing any prices in our pricing DB, so looks like this might be an issue in https://github.com/infracost/cloud-pricing-api. We will look at getting a fix in for the next release.

```
> SELECT prices FROM products where "vendorName"='gcp' AND "service"='Compute Engine' AND "productFamily"='Compute Instance'  AND "region"='us-central1' AND "attributes"->>'machineType' LIKE 'g2-standard%';
 prices
--------
(0 rows)
```

**Comment by aliscott** [comment]
Thanks for reporting this @KivraChristoffer. This has now been fixed and released in [v0.3.15 of the Cloud Pricing API](https://github.com/infracost/cloud-pricing-api/releases/tag/v0.3.15)
